### PR TITLE
Update LRUCache.js

### PR DESCRIPTION
**Pull request by awojobigodfrey**
Added a clear function for cache

# Welcome to the JavaScript community

[Open in Gitpod](https://gitpod.io/from-referrer/)&nbsp;[know more](https://www.gitpod.io/docs/pull-requests/)

### Describe your change:

- [ ] Add an algorithm?
- [x] Fix a bug or typo in an existing algorithm?
- [ ] Documentation change?

### Checklist:

- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Javascript/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized.
- [x] I know that pull requests will not be merged if they fail the automated tests.
- [ ] This PR only changes one algorithm file. To ease review, please open separate PRs for separate algorithms.
- [ ] All new JavaScript files are placed inside an existing directory.
- [ ] All filenames should use the UpperCamelCase (PascalCase) style. There should be no spaces in filenames.
      **Example:**`UserProfile.js` is allowed but `userprofile.js`,`Userprofile.js`,`user-Profile.js`,`userProfile.js` are not
- [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
- [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: Cache/LRUCache.js [hunk 1]
```diff
@@ -83,6 +83,11 @@ class LRUCache {
     this.cache.set(key, value)
   }
 
+  clear () {
+      this.cache.clear()
+      return this
```
Comment (appgurueu) [inline Cache/LRUCache.js:88]: Why?

**Review by appgurueu — CHANGES_REQUESTED** [review]
Please - at the very least - fix your indentation. Why is this feature necessary? Why can't I just call `obj.cache.clear()`?

**Comment by raklaptudirm** [comment]
`obj.clear()` is less clear then `obj.cache.clear()` imo.

File: Cache/LRUCache.js [hunk 2]
```diff
@@ -83,6 +83,11 @@ class LRUCache {
     this.cache.set(key, value)
   }
 
+  clear () {
+    this.cache.clear()
+    return this
```
Comment (CarlosZoft) [inline Cache/LRUCache.js:88]: Why returns "this"?

**Comment by raklaptudirm** [comment]
@appgurueu should I close this? We both seem to think it is unnecessary and less readable.

**Pull request closed without merging**